docs/reference.rst

```diff
@@ -98,6 +99,8 @@ Example configuration:
   be stored (e.g. slot specific metadata, see :ref:`slot-status`).
   This file should be located on a filesystem which is not overwritten during
   updates.
+  This method is preferable over per-slots status files as it allows to store
+  data also for read-only or (temporary) filesystem-less slots.
```

Comment:
Not 100% sure, but for me it feels better to have a replacement ``s/temporary/temporarily/``.

Reply:
I don't see that one should use an adverb here.

Reply:
Perhaps I got the intended meaning wrong: I thought that it was bound to ``filesystem-less``.